unstake.js

```diff
@@ -111,6 +111,26 @@ const createRedeemRequest = async () => {
 }
 // createRedeemRequest();
 
+
+const checkRedeemEvent = async () => {
+    //insert secret here below. Replace Goerli with any other
+    const provider = new ethers.providers.WebSocketProvider(
+        `wss://goerli.infura.io/ws/v3/${INFURA_SECRET}`, "goerli"
+      );
+    
+    const redeemManagerContract = new ethers.Contract(CONTRACT_ADDRESS, ABI, provider.getSigner());
```

Comment:
We will need to add RedeemManager contract address here. 0x080b3a41390b357Ad7e8097644d1DEDf57AD3375 and ABI https://etherscan.io/address/0x423ce5282c460eed5fe0786b4d47d2c2a4ef3721#code

Reply:
ah this is a different contract ABI than the River one for the other contract calls, correct?

Reply:
yup thats right

Reply:
@josh-alluvial do you have the goerli one?

Reply:
yup: 

Contract: https://goerli.etherscan.io/address/0x0693875efbF04dDAd955c04332bA3324472DF980
Implementation: https://goerli.etherscan.io/address/0x653b549554669a06bb9e260b9f2c0a54f5d7e722#code